src/gameplay_v2/roles/components/hideFromMerlin.ts

```diff
@@ -0,0 +1,7 @@
+import Component from './component';
+
+//
+export class hideFromMerlinC extends Component {
+  name = 'SeeSpies';
+  data = {};
+}
```

Comment:
Nah, don't need this. I presume you mean this for Oberon yea? This logic should go in `seeSpies`'s system.